lib/Elevate/Blockers/Databases.pm

```diff
@@ -111,9 +111,31 @@ sub _blocker_old_mysql ($self) {
     return $mysql_is_provided_by_cloudlinux ? $self->_blocker_old_cloudlinux_mysql() : $self->_blocker_old_cpanel_mysql();
 }
 
-# TODO: RE-234 implement this
 sub _blocker_old_cloudlinux_mysql ($self) {
-    return;
+    my $cloudlinux_database_info = cpev::read_stage_file('cloudlinux_database_info');
+    my $db_version               = $cloudlinux_database_info->{db_version};
+
+    # 5.5 gets stored as 55 and so on and so forth since there are no .'s
+    # for the version in the RPM package name
+    if ( $db_version < 55 ) {
+        my $pretty_distro_name = $self->upgrade_to_pretty_name();
+        my $db_dot_version = $db_version / 10;
```

Comment:
dividing an integer by 10 doesn't always lead to a neat decimal. Sometimes it could be 5.09999999. It's sometimes better to do `s/^5/5./;`

Reply:
I didn't see this pop up in my testing, but I went ahead and made this change just in case.  This is why I avoid floating numbers whenever possible.

Reply:
Math::BigFloat will solve this if you have to.